### PR TITLE
MRG: Better split file handling in Report.parse_folder()

### PR DESCRIPTION
**Pull request by hoechenberger**
Better split file handling in Reports: don't "report" on any splits but the first.

cc @agramfort, wdyt?

### CODE REVIEW

File: mne/report.py [hunk 1]
```diff
@@ -1456,6 +1456,26 @@ class construction.
...
 
+        # For split files, only keep the first one.
+        fnames_to_remove = []
+        for fname in fnames:
+            if _endswith(fname, ('raw', 'sss', 'meg')):
+                data = read_raw_fif(fname, allow_maxshield=True, preload=False)
+            elif _endswith(fname, ('epo',)):
+                data = read_epochs(fname, preload=False)
+            elif _endswith(fname, ('ave',)):
+                data = read_evokeds(fname, condition=0, allow_maxshield=True)
```
Comment (agramfort) [inline mne/report.py:1467]: can we call `data` `inst` ?

Comment (hoechenberger) [reply mne/report.py]: Done

**Comment by hoechenberger** [comment]
@agramfort can you check if this fixes your issue?

File: doc/changes/latest.inc [hunk 2]
```diff
@@ -103,6 +103,8 @@ Bugs
 
 - Attempting to remove baseline correction from preloaded `~mne.Epochs` will now raise an exception by `Richard Höchenberger`_ (:gh:`8435`)
 
+- :meth:`mne.Report.parse_folder` will now correctly handle split FIFF files by `Richard Höchenberger`_ (:gh:`8486`)
```
Comment (agramfort) [inline doc/changes/latest.inc:106]:
```suggestion
- :meth:`mne.Report.parse_folder` will now correctly handles split FIFF files by `Richard Höchenberger`_ (:gh:`8486`)
```

**Review by agramfort — APPROVED** [review]
thx @hoechenberger !

File: doc/changes/latest.inc (hunk 2 above)
Comment (drammock) [reply doc/changes/latest.inc:106]: I think it is correct the way it is already (present tense *handles*, future tense *will handle*).

**Comment by agramfort** [comment]
thx @hoechenberger

**Comment by larsoner** [comment]
This broke CircleCI:

https://app.circleci.com/pipelines/github/mne-tools/mne-python/4556/workflows/229c2f13-d8c1-460e-b964-682d94c01ea9/jobs/23290

Can you look @hoechenberger ?

**Pull request merged**